net-misc/tigervnc/tigervnc-1.6.0-r2.ebuild

```diff
@@ -38,7 +38,11 @@ RDEPEND="virtual/jpeg:0
 		x11-apps/xsetroot
 		>=x11-misc/xkeyboard-config-2.4.1-r3
 		opengl? ( >=app-eselect/eselect-opengl-1.3.1-r1 )
-		xorgmodule? ( =x11-base/xorg-server-${XSERVER_VERSION%.*}* )
+		xorgmodule? (
+			=x11-base/xorg-server-${XSERVER_VERSION%.*}*
+			drm? ( >=x11-libs/libdrm-2.4.26 )
+		)
+		drm? ( x11-libs/libdrm )
```

Comment:
Initially, I thought that no, it can't be simplified.
(`configure` script requires different versions of libdrm in different cases: `>= 2.3.0` or `>= 2.4.46`)

But I double-checked that, and I think, I got this wrong.
So, yeah. you're right.
It should be just

     + drm? ( x11-libs/libdrm )